meta/main.yml

```diff
@@ -3,13 +3,12 @@
 dependencies: [ ]
 
 galaxy_info:
-  author: "David Reed"
   description: "Install Rsyslog."
   license: MIT
-  min_ansible_version: 1.9
+  min_ansible_version: 2.0
   platforms:
     - name: Ubuntu
       versions:
-        - all
+        - trusty
```

Comment:
will this cause an install on xenial to fail, or is it just advisory?
we are using the startup scripts included in the deb, so presumably it should work ok on xenial?

Reply:
It's just advisory, if this works on Xenial then I can add it to the list.

Reply:
Added Xenial